frontend/src/scenes/plugins/tabs/installed/sections/EnabledPluginsSection.tsx

```diff
@@ -0,0 +1,244 @@
+import {
+    CaretRightOutlined,
+    CaretDownOutlined,
+    CloseOutlined,
+    SaveOutlined,
+    OrderedListOutlined,
+} from '@ant-design/icons'
+import { Button, Col, Row, Space, Tag, Tooltip } from 'antd'
+import { Subtitle } from 'lib/components/PageHeader'
+import React from 'react'
+import { useActions, useValues } from 'kea'
+import { PluginSection, pluginsLogic } from 'scenes/plugins/pluginsLogic'
+import { InstalledPlugin } from '../InstalledPlugin'
+import { canConfigurePlugins } from '../../../access'
+import { userLogic } from 'scenes/userLogic'
+import { SortableContainer, SortableElement, SortableHandle } from 'react-sortable-hoc'
+import { PluginTypeWithConfig } from 'scenes/plugins/types'
+
+type HandleProps = { children?: JSX.Element }
+
+const DragColumn = SortableHandle<HandleProps>(({ children }: HandleProps) => (
+    <Col className="order-handle">{children}</Col>
+))
+
+const SortablePlugin = SortableElement(
+    ({
+        plugin,
+        order,
+        maxOrder,
+        rearranging,
+    }: {
+        plugin: PluginTypeWithConfig
+        order: number
+        maxOrder: number
+        rearranging: boolean
+    }) => (
+        <InstalledPlugin
+            plugin={plugin}
+            order={order}
+            maxOrder={maxOrder}
+            rearranging={rearranging}
+            DragColumn={DragColumn}
+        />
+    )
+)
+const SortablePlugins = SortableContainer(({ children }: { children: React.ReactNode }) => {
+    return (
+        <Row gutter={16} style={{ marginTop: 16 }}>
+            {children}
+        </Row>
+    )
+})
+
+export function EnabledPluginSection(): JSX.Element {
+    const { user } = useValues(userLogic)
+
+    const {
+        rearrange,
+        setTemporaryOrder,
+        cancelRearranging,
+        savePluginOrders,
+        makePluginOrderSaveable,
+        toggleSectionOpen,
+    } = useActions(pluginsLogic)
+
+    const {
+        enabledPlugins,
+        filteredEnabledPlugins,
+        sortableEnabledPlugins,
+        unsortableEnabledPlugins,
+        rearranging,
+        loading,
+        temporaryOrder,
+        pluginOrderSaveable,
+        searchTerm,
+        sectionsOpen,
+    } = useValues(pluginsLogic)
+
+    const canRearrange: boolean = canConfigurePlugins(user?.organization) && sortableEnabledPlugins.length > 1
+
+    const rearrangingButtons = rearranging ? (
+        <>
+            <Button
+                type="primary"
+                icon={<SaveOutlined />}
+                loading={loading}
+                onClick={(e) => {
+                    e.stopPropagation()
+                    savePluginOrders(temporaryOrder)
+                }}
+                disabled={!pluginOrderSaveable}
+            >
+                Save order
+            </Button>
+            <Button type="default" icon={<CloseOutlined />} onClick={cancelRearranging}>
+                Cancel
+            </Button>
```

Comment:
Kinda weird that clicking this causes the "Enabled plugins" section to close.

Reply:
Ah yes need to stop event propagation here probably